src/components/Faq/Faq.tsx

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
-import { Collapse } from 'antd';
+import { Collapse, CollapseProps } from 'antd';
 import classNames from 'classnames';
 
-import ArrowIcon from '@svg/arrow.inline.svg';
-import { iconsCommon } from '@utils/imageSource';
-import { DOCUMENTATION_URL } from '@utils/constants';
-import { createBemBlockBuilder } from '@utils';
+import { Link } from '@app/components/Link';
+import { createBemBlockBuilder, DOCUMENTATION_URL, iconsCommon } from '@app/utils';
 
-import { Link } from '../Link';
+import ArrowIcon from '../../svg/arrow.inline.svg';
 
 import './Faq.scss';
 
+interface Props {
+  items: CollapseProps['items'];
+  titleId?: string;
+  documentationLink?: string;
+  showMoreInfoLink: boolean;
```

Comment:
Looks like this prop is also optional.
```suggestion
  showMoreInfoLink?: boolean;
```